src/main/java/org/carlmontrobotics/Constants.java

```diff
@@ -19,26 +24,71 @@ public static final class Drivetrain {
 	}
 	
 	public static final class Arm {
```

Comment:
Please reorganize arm constants and put them in the following order:
- ports
- config for motors (inverted, encoder offset, etc)
- goal positions
- feedforward, pid, trapezoid constants
- Boundaries (arm cannot go beyond certain angles)
- Controller buttons should go in OI not Arm